MEME/MEME/Model/ModelHomeView/View/ModelHomeViewController.swift

```diff
@@ -0,0 +1,403 @@
+            make.trailing.equalTo(contentsView.snp.trailing).offset(-24)
+        }
+        collectionView.snp.makeConstraints {make in
+            make.top.equalTo(viewAllReservationsButton.snp.bottom).offset(13)
+            make.height.equalTo(800)
+            make.leading.equalTo(contentsView.snp.leading).offset(24)
+            make.trailing.equalTo(contentsView.snp.trailing).offset(-24)
+            make.bottom.equalTo(contentsView.snp.bottom)
+    
+        }
+    }
+    
+    // MARK: - BindViewModel
+    private func bindViewModel() {
+        let memberId = KeyChainManager.loadMemberID()
+        
+        let input = ModelHomeViewModel.Input(
+            modelReservationsTrigger: Observable.just(memberId),
+            getRecommendArtistByReviewTrigger: .just(()),
+            getRecommendArtistByRecentTrigger: .just(())
+        )
+        
+        let output = viewModel.transform(input: input)
+        
+        Observable.combineLatest(
+            output.modelReservations,
+            output.recommendArtistByReview,
+            output.recommendArtistByRecent
+        ).observe(on: MainScheduler.instance)
+        .subscribe(onNext: { [weak self] 
+            modelReservations,
+            recommendArtistByReview,
+            recommendArtistByRecent in
+            self?.setSnapShot(
+                modelReservations: modelReservations,
+                recommendArtistByReview: recommendArtistByReview,
+                recommendArtistByRecent: recommendArtistByRecent
+            )
+        }).disposed(by: disposeBag)
+    }
+    //MARK: -Actions
+    func setUserNickName() {
+        if let nickname = KeyChainManager.read(forkey: .nickName) {
+            modelWelcomeLabel.text = "\(nickname)님, 환영합니다!"
+        } else {
+            modelWelcomeLabel.text = "환영합니다!"
+        }
+    }
+    
+    func setupSearchBar() {
+        searchMakeup.delegate = self
+    }
+    @objc private func viewAllReservationsTapped() {
+        let reservationsVC = ManagementReservationsViewController()
+        reservationsVC.hidesBottomBarWhenPushed = true
+        navigationController?.pushViewController(reservationsVC, animated: true)
+    }
+    
+    //MARK: -CollectionView
+    private func setupCollectionView() {
+        collectionView.delegate = self
+        
+        collectionView.register(ModelNonReservationViewCell.self,
+                                forCellWithReuseIdentifier: ModelNonReservationViewCell.identifier)
+        collectionView.register(UINib(nibName: "ModelReservationConfirmViewCell", bundle: nil),
+                                forCellWithReuseIdentifier: ModelReservationConfirmViewCell.identifier)
+        collectionView.register(UINib(nibName: "SelectMakeupCardViewCell", bundle: nil),
+                                forCellWithReuseIdentifier: SelectMakeupCardViewCell.identifier)
+        collectionView.register(UINib(nibName: "SelectMakeupCardViewCell", bundle: nil),
```

Comment:
```suggestion
        collectionView.register(UINib(nibName: ModelReservationConfirmViewCell.className, bundle: nil),
                                forCellWithReuseIdentifier: ModelReservationConfirmViewCell.identifier)
        collectionView.register(UINib(nibName: SelectMakeupCardViewCell.className, bundle: nil),
                                forCellWithReuseIdentifier: SelectMakeupCardViewCell.identifier)
        collectionView.register(UINib(nibName: SelectMakeupCardViewCell.className, bundle: nil),
```
과 같이 사용하여 하드 코딩은 지양하는 것이 좋을 것 같습니다!